video_store_API_stuff.py

```diff
@@ -0,0 +1,164 @@
+import requests
+
+
+class Video:
+    def __init__(self, url="http://localhost:5000", selected_video=None):
+        self.url = url
+        self.selected_video = selected_video
+
+
+    def add_video(self, title="The Void", release_date=None, total_inventory=None):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
+            "total_inventory": total_inventory
+        }
+        response = requests.post(self.url+"/videos",json=query_params)
+        return response.json()
```

Comment:
One opportunity for improvement for many people for this project is handling error codes from the API requests. In general, when writing code to call an API, it's appropriate to check whether the request was successful or it triggered some sort of error. This is where HTTP response codes come in handy! We can usually assume that a 2XX response code means there was no error and any other response code means there was some sort of error. It's very common to have an if...elif...else chain to handle different ranges of response codes.
This applies to all the other API calls as well.